jenkins_pipelines/scripts/kafka/consumer.py

```diff
@@ -0,0 +1,205 @@
+            custom_repositories_formatted = json.dumps(
+                {
+                    key: value for key, value in json.load(custom_repositories).items()
+                    if key in instances_involved
+                },
+                indent=4
+            )
+        build_parameters = {
+            'cucumber_gitrepo': 'https://github.com/SUSE/spacewalk.git',
+            'cucumber_ref': 'Manager-4.3',
+            'tf_file': 'susemanager-ci/terracumber_config/tf_files/SUSEManager-4.3-SLE-update.tf',
+            'sumaform_gitrepo': 'https://github.com/uyuni-project/sumaform.git',
+            'sumaform_ref': 'master',
+            'sumaform_backend': 'libvirt',
+            'terraform_bin': '/usr/bin/terraform',
+            'terraform_bin_plugins': '/usr/bin',
+            'terraform_parallelism': '',
+            'terracumber_gitrepo': 'https://github.com/uyuni-project/terracumber.git',
+            'terracumber_ref': 'master',
+            'minions_to_run': 'sle15sp4_minion',
+            'use_previous_terraform_state': 'false',
+            'must_deploy': 'true',
+            'must_run_core': 'true',
+            'must_sync': 'true',
+            'enable_proxy_stages': 'true',
+            'enable_client_stages': 'true',
+            'must_add_MU_repositories': 'true',
+            'must_add_non_MU_repositories': 'true',
+            'must_add_keys': 'true',
+            'must_create_bootstrap_repos': 'true',
+            'must_boot_node': 'true',
+            'must_run_tests': 'true',
+            'must_run_containerization_tests': 'false',
+            'confirm_before_continue': 'false',
+            'custom_repositories': custom_repositories_formatted
+        }
+
+        request = self.api_clients.post(
+            'https://ci.suse.de/job/manager-4.3-qe-sle-update/buildWithParameters',
+            params=build_parameters
+        )
+
+        if request.status_code == 201:
+            time.sleep(10)  # to avoid "In the quiet period. Expires in <10 sec"
+            request = self.api_clients.get(f"{request.headers['Location']}/api/json")
+            response = request.json()
+            try:
+                build_number = response['executable']['number']
+                os.rename('custom_repositories.json', f'custom_repositories_{build_number}.json')
+                return build_number
+            except KeyError:
+                logging.error(f"Build number {build_number} was not found in the currently running pipelines, latest output: {response['why']}. Please check if someone else is working on the pipeline.")
+                return None
+        if request.status_code == 431:
+            logging.error(f'Request is too big, perhaps too many RRs to be accepted generated big JSON, run pipeline manually using generated custom_repositories.json in {os.getcwd()}')
+        return None
+
+    def pipeline_status(self):
```

Comment:
Nitpicky, but maybe is worth changing this method name and signature to something like :

```python
def pipeline_enabled(self) -> bool:
```

Reply:
done